samples/matter/temperature_sensor/src/app_task.h

```diff
@@ -36,7 +36,9 @@ class AppTask {
 		}
 	}
 
-	int16_t GetCurrentTemperature() const { return mCurrentTemperature; }
+	/* Defined by cluster temperature measured value = 100 x temperature in degC with resolution of
+	 * 0.01 degC. */
+	int16_t GetCurrentTemperature() const { return mCurrentTemperature * 100; }
```

Comment:
I'm wondering about the rest of the configuration. We have also mTemperatureSensorMinValue and mTemperatureSensorMaxValue that are not multiplied by 100, which is probably also incorrect? The verification scripts will for sure validate than measured value is not out of the min-max bounds and fail because of that.

I think we should update min-max values in zap and also store all the values as already multiplied by 100, instead of converting it during getter call.

Reply:
Done